react-components/src/stories/CogniteCadModelContainer.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { CogniteCadModelContainer, RevealContainer } from '..';
+import { CogniteClient } from '@cognite/sdk';
+import { Color, Matrix4 } from 'three';
+
+const meta = {
+  title: 'Example/CogniteCadModelContainer',
+  component: CogniteCadModelContainer,
+  tags: ['autodocs']
+} satisfies Meta<typeof CogniteCadModelContainer>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+const token = new URLSearchParams(window.location.search).get('token')!;
```

Comment:
Would at least like to have a big TODO here, to make auth more implicit, along the lines of examples :sweat_smile: